s30161-pj-NikodemCyrzan/zajecia1/instalacja-debiana-linux.md

```diff
@@ -0,0 +1,523 @@
+
+#### Tabela obrazująca niektóre z opcji:
+
+| Tryb                    | VM -> host | VM <- host             | VM1 <-> VM2 | VM -> LAN | VM <- LAN              |
+| ----------------------- | ---------- | ---------------------- | ----------- | --------- | ---------------------- |
+| Niepodłączona           | -          | -                      | -           | -         | -                      |
+| NAT                     | +          | poprzez przekierowanie | -           | +         | poprzez przekierowanie |
+| Sieć NAT                | +          | poprzez przekierowanie | +           | +         | poprzez przekierowanie |
+| Sieć wewnętrzna         | -          | -                      | +           | -         | -                      |
+| Sieć mostkowana         | +          | +                      | +           | +         | +                      |
+| Karta sieci izolowaniej | +          | +                      | +           | -         | -                      |
+
+> Mój wybór
+>
+> `NAT`, aby mieć dostęp do internetu podczas procesu instalacji.
+
+## 3. Instalacja systemu
+
+### Menu instalatora
+
+#### Graphical install
+
+Instalacja z pomocą interfejsu graficznego.
+
+#### Install
+
+Instalacja tekstowa. Nie wymaga myszki i jest przydatna w przypadku maszyn które mają ograniczone zasoby na tyle, że uruchomienie GUI byłoby dla nich ciężkie, albo niemożliwe.
+
+> Mój wybór
+>
+> Ponieważ na klawiaturze konfiguruje się szybciej i wygodniej.
+
+#### Opcje zaawansowane
+
+Dodatkowe opcje umożliwiające na przykład automatyczne zainstalowanie systemu.
+
+#### Accessible dark contrast installer menu
+
+Instalacja z wysokim kontrastem dla osób niedowidzących.
+
+#### Install with speech synthesis
+
+Instalacja z syntezatorem mowy dla osób niewidomych. Wymaga karty dźwiękowej.
+
+### Wybór języka
+
+> Mój wybór
+>
+> `English`, ponieważ w internecie prościej uzyskać pomoc w tym języku, niż w języku polskim.
+
+### Wybór lokalizacji
+
+> Mój wybór
+>
+> `Poland`, ponieważ umożliwia to poprawnie ustawić źródło z którego pobierane będą pakiety.
+
+### Wybór kodowania znaków
+
+> Mój wybór
+>
+> `en_US.UTF-8`, ponieważ jest to najbezpieczniejszy format kodowania znaków, który nie umożliwia wpisywania znaków diakrytycznych, a więc nie pojawią się w przyszłości z tego powodu problemy z odczytywaniem plików.
+
+### Ustawienia klawiatury
+
+> Mój wybór
+>
+> `American English`, ponieważ jest to układ większości klawiatór w Polsce.
+
+### Nadawanie nazwy hosta
+
+Jest to nazwa pod którą inne urządzenia w sieci będą widziały ten system.
+
+> Mój wybór
+>
+> `debian12`
+
+### Partycjonowanie
+
+#### Metody:
+
+-   Guided - use entire disk
+-   Guided - use entire disk and set up LVM
+-   Guided - use entire disk and set up encrypted LVM
+-   Manual
+
+> Mój wybór
+>
+> `Guided - use entire disk`, ponieważ pierwszy raz instaluję ten system i chcę przeprowadzić partycjonowanie z przewodnikiem i bez pomocy LVM.
```

Comment:
Przwodnik to pomoc. Nie LVM
LVM jest do czego innego.
Natomiast treścią zadania jest opisanie wszystkich możliwych opcji a nie tylko tych, które się wybiera. Także opis RAIDu jak i LVM-a czy szyfrowanego LVM-a trzeba zrobić.